--- install.rdf	2014-07-05 16:18:22.000000000 +0200
+++ install-localized.rdf	2014-07-05 16:21:56.176424545 +0200
@@ -19,10 +19,21 @@
 
     <!-- Front End MetaData -->
     <em:name>VoIPGRID</em:name>
-    <em:description>https://bugzilla.mozilla.org/show_bug.cgi?id=661083. Replace this description with the localized description in install.rdf manually! Run cfx xpi, unzip voipgrid.xpi, edit install.rdf and re-zip or just run ./localize_xpi.sh.</em:description>
+    <em:localized>
+      <Description>
+        <em:locale>en-US</em:locale>
+        <em:description>Firefox add-on for click-to-dial, status updates and queue monitoring.</em:description>
+      </Description>
+    </em:localized>
+    <em:localized>
+      <Description>
+        <em:locale>nl-NL</em:locale>
+        <em:description>Firefox plugin voor klik&amp;bel, statusupdate en wachtrij monitoring.</em:description>
+      </Description>
+    </em:localized>
     <em:creator>VoIPGRID</em:creator>
     
     <em:optionsType>2</em:optionsType>
     
   </Description>
-</RDF>
\ No newline at end of file
+</RDF>
